sale_credit_point/models/point_history.py

```diff
@@ -34,3 +35,14 @@ class PointHistory(models.Model):
     comment = fields.Char(
         string="Comment"
     )
+    current_year = fields.Boolean(
+        compute="_compute_current_year",
+        store=True,
+    )
+
+    @api.depends('create_date')
```

Comment:
so this is going to work only on create.... :)
You don't need this field. 1st example I grepped in odoo code:
```xml
<filter string="This Year" name="year" domain="[('create_date','&lt;=', time.strftime('%%Y-12-31')),('create_date','&gt;=',time.strftime('%%Y-01-01'))]"/>
```